internal/proto/controller/api/resources/targets/v1/target.proto

```diff
@@ -158,8 +159,19 @@ message Target {
   // Output only. The egress credential sources associated with this Target.
   repeated CredentialSource egress_credential_sources = 510 [json_name = "egress_credential_sources"];
 
-  // The attributes that are applicable for the specific Target.
-  google.protobuf.Struct attributes = 200 [(custom_options.v1.generate_sdk_option) = true];
+  oneof attrs {
```

Comment:
Not that our comments have been great in the protos, even the previous comment didn't give much: `The attributes that are applicable for the specific Target.`.
But it might be useful to have something more than the auto generated `oneof` since I am not sure how clear `attrs` is:
```
// Types that are assignable to Attrs:
	//	*Target_Attributes
	//	*Target_TcpTargetAttributes
```

This is a minor nit and feel free to drop this for later

Reply:
I agree that we should look at these comments. But there is a particular consideration with these in how they relate to the openapi/swagger output. These comments are used in the generated swagger doc, except when a field has `(google.api.field_visibility).restriction = "INTERNAL"`, so in the case of the `TcpTargetAttributes` mentioned in your other comment, any comments would be dropped from the swagger doc, which is intentional. 

As for the Attrs field itself in the generated go code, I found the auto generated doc useful enough to direct a developer to the structs that could be in the oneof, which is think is sufficient *if* those structs are well documented.